herepy/geocoder_api.py

```diff
@@ -106,8 +106,8 @@ def address_with_boundingbox(
 
     def address_with_details(
         self,
-        house_number: int,
-        street: str,
+        house_number: Optional[int] = None,
+        street: Optional[str] = None,
```

Comment:
Add the optional parameters after the required parameters to fix the linting issues